composer.json

```
@@ -12,9 +12,9 @@
     ],
     "require": {
         "php": "^7.2",
-        "symfony/console": "^4.0",
-        "symfony/process": "^4.0",
-        "themsaid/forge-sdk": "^1.2",
+        "symfony/console": "^5.0",
+        "symfony/process": "^5.0",
```

```suggestion
        "symfony/process": "^4.0 || ^5.0",
```

I'd rather keep these like so to still _allow_ for `4.x` to be installed. Otherwise we'll get conflicts with people who don't have the latest version of Homestead 🙂